.github/workflows/nightfalldlp.yml

```diff
@@ -0,0 +1,18 @@
+name: nightfalldlp
+on: [pull_request]
```

Comment:
Do we want to change this to 
```
on:
  push:
    branches:
      - master
  pull_request:
```
so the check also runs on direct pushes to master?

Reply:
sure!